### PR TITLE
Fixes #18174: commands from error message are launched due to backticks

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/18174

### CODE REVIEW

File: share/commands/agent-inventory [hunk 1]
```diff
@@ -34,7 +34,7 @@ bootstrap_check
 # We cannnot start an inventory in bootstrap policies
 if cmp -s "/var/rudder/cfengine-community/inputs/failsafe.cf" "/opt/rudder/share/bootstrap-promises/failsafe.cf"
 then
-  >&2 echo "Agent is currently in bootstrap policies, cannot run an inventory. Please download initial policies from the server with command `rudder agent update` first. If problem persists, use `rudder agent check` for diagnostic"
+  >&2 echo "Agent is currently in bootstrap policies, cannot run an inventory. Please download initial policies from the server with command \`rudder agent update\` first. If problem persists, use \`rudder agent check\` for diagnostic"
```
Comment (peckpeck) [inline share/commands/agent-inventory:37]: just don't use backticks in shell code and put simple quotes instead

**Review by peckpeck — CHANGES_REQUESTED** [review]

File: share/commands/agent-inventory (hunk 1 above)
Comment (fanf) [reply share/commands/agent-inventory:37]: I agree, it's less dangerous

**Comment by VinceMacBuche** [comment]
PR updated with a new commit

**Review by peckpeck — APPROVED** [review]

**Comment by VinceMacBuche** [comment]
OK, squash merging this PR

**Pull request merged**